azure/components/baseline/setup.ftl

```diff
@@ -320,6 +328,33 @@
       [/#if]
     [/#list]
 
+    [#if deploymentSubsetRequired("prologue", false)]
+      [#list keyVaultAdmins as adminGrp ]
+        [#local adminId = formatId(keyvaultId,adminGrp)]
+        [@addToDefaultBashScriptOutput
+          content=
+          [
+            "case $\{DEPLOYMENT_OPERATION} in",
+            "  create|update)",
+            "    ADMINGRP=$(az role definition list --name "+adminGrp+")",
+            "    if [[ $\{#ADMINGRP[@]} -eq 0 ]] ; then",
+            "      fatal \"Azure Administrator role does not exist: "+adminGrp+"\"",
+            "    else"
```

Comment:
That should end with `fi` instead of if and also we should exit after the fatal if none of the other steps can be run

Reply:
The `fi` is at L350 to make the pseudoArmStackOutputScript() conditional, or are you talking about elsewhere?